docs/SOP.md

```diff
@@ -100,6 +101,17 @@ In general, the metadata record of an ontology in the OBO Foundry metadata regis
 
 For a discussion on this SOP, see [here](https://github.com/OBOFoundry/OBOFoundry.github.io/issues/1848).
 
+<a name="REACTIVATION"></a>
+## Reviving obsolete, orphaned, or inactive ontologies
+Ontologies that became dormant and were marked as either obsolete, orphaned, or inactive require a review process if they want to be marked as active again.
+Currently, these are the required steps:
+1. If an ontology is desired to be reactivated, a member of the ontology community first makes a PR on the metadata file with the desired change in state.
```

Comment:
showing exceptional community engagement is a great point. Aligning with forward looking OBO community concepts (e.g., aligning with COB, adding non-required high quality metadata like root terms, etc.) would also show that there is good faith in the people who want to revive the ontology

Reply:
Lets separate these concerns. I 100% agree with you, but if we tie COB alignment and passing dashboard etc to curations like this one we will just create frustration without solution. I think we need to work out more carrots than sticks to get this done.